### PR TITLE
Modify post-merge script to reflect 5.0.0 build changes

### PR DESCRIPTION
**Pull request by TwizzyDizzy**
This fixes the post-merge script to reflect build changes needed in 5.0.0.

As discussed in https://github.com/LycheeOrg/Lychee/issues/2104

Cheers
Thomas

### CODE REVIEW

File: scripts/post-merge [hunk 1]
```diff
@@ -29,6 +29,17 @@ else
...
         composer install --no-dev --prefer-dist
+        if command -v npm > /dev/null; then
+            printf "\n${ORANGE}npm detected${NO_COLOR}\n"
+            if  [ -f "package.json" ]; then
+                echo "npm install --no-audit --no-fund"
+                npm install --no-audit --no-fund
+                echo "npm run build"
+                npm run build
+            else
+                printf "${GREEN}no package.json found${NO_COLOR}\n"
```
Comment (qwerty287) [inline scripts/post-merge]: You can remove that `else` branch.

Comment (TwizzyDizzy) [reply scripts/post-merge]: I just jumped back to one commit before the "if -f package.json" was added and back then, it indeed seems to not have been part of the repo at this point.

So I'm getting you right in that you want to remove this condition due to the fact that `package.json` is now part of the repo, correct?

If that's the case, then I agree. And I would then add, that line 21-26 (`master`) can be adjusted in the same way.

Cheers
Thomas

Comment (ildyria) [reply scripts/post-merge]: > So I'm getting you right in that you want to remove this condition due to the fact that package.json is now part of the repo, correct?

yes.

> If that's the case, then I agree. And I would then add, that line 21-26 (master) can be adjusted in the same way.

Indeed.

**Review by ildyria — APPROVED** [review]
LGTM.

**Pull request merged**